riff-raff/app/conf/context.scala

```diff
@@ -159,8 +163,16 @@ class Configuration(val application: String, val webappConfDirectory: String = "
       implicit lazy val bucketName = configuration.getStringProperty("tag.aws.bucketName")
       lazy val accessKey = configuration.getStringProperty("tag.aws.accessKey")
       lazy val secretKey = configuration.getStringProperty("tag.aws.secretKey")
-      implicit lazy val client = new AmazonS3Client(credentialsProvider)
       lazy val credentialsProvider = credentialsProviderChain(accessKey, secretKey)
+      lazy val regionName = configuration.getStringProperty("tag.aws.region", "eu-west-1")
+      lazy val region = awsRegion(regionName)
+      implicit lazy val client = new AmazonS3Client(credentialsProvider).withRegion(region)
+    }
+  }
+
+  object target {
+    object aws {
+      lazy val defaultRegionName = configuration.getStringProperty("target.aws.defaultRegion", "eu-west-1")
```

Comment:
There is no field for region in deploy.json, but open to making it mandatory in riff-raff.yaml.